### DISCUSSION TITLE
Laragon vs Xampp

### DISCUSSION THREAD

**Discussion by tiomanisland** [leokhoa/laragon#927] [General]
After years of using Xampp it started to crash under Windows 11. It is just terrible. Randomly MySQL won't start and my databases are lost or if I can recover them they are corrupted. For days or weeks everything is ok, then just suddently MySQL reports error and doesn't start anymore. I tried everything to avoid this (settings, run as admin, etc.) nothing helps. Today I installed Laragon and I want to see if MySQL under Laragon will crash also. What is your experience?

**Comment by gmaniel** [comment]
Hello
I have been using Laragon for over 4 years, I have used it on Windows 10 and Windows 11, I have not had any problems so far, I even have mysql and mariadb installed and both run without problem, as Laragon is based on Wamp Server, its compatibility is very good with Windows.

I hope my comment is useful to you.

Greetings

**Comment by tiomanisland** [comment]
That's good to hear! So even if you didn't stop webserver and MySQL when restarting/hibernate Windows, MySQL never crashed?

**Comment by Pen-y-Fan** [comment]
I can confirm I run Laragon for long periods of time with no problems. Restarting Windows is not a problem either. Laragon can auto-launch and auto-start, if required, which can be set in the settings. I regularly put Windows to Sleep, Laragon is happy when Windows does that too.

If you are setting up Laragon for the first time, my blog may help [how to set up Laragon on a new Windows computer (part 1) - Install Laragon](https://pen-y-fan.github.io/2022/11/07/how-to-set-up-laragon-on-a-new-windows-computer/). If you want to change PHP, Apache or other components, see the Wiki:

- [[Tutorial] How to add another Apache version](https://github.com/leokhoa/laragon/wiki/%5BTutorial%5D-How-to-add-another-Apache-version)
- [[Tutorial] How to add another PHP version (PHP 8.3.4)](https://github.com/leokhoa/laragon/wiki/%5BTutorial%5D-How-to-add-another-PHP-version-(PHP-8.3.4))

The next version of Largaon is expected next week too.
